nano/node/node.cpp

```diff
@@ -968,8 +974,13 @@ namespace nano
 std::unique_ptr<seq_con_info_component> collect_seq_con_info (alarm & alarm, const std::string & name)
 {
 	auto composite = std::make_unique<seq_con_info_composite> (name);
-	auto count = alarm.operations.size ();
-	auto sizeof_element = sizeof (decltype (alarm.operations)::value_type);
+	size_t count = 0;
+	size_t sizeof_element = 0;
+	{
+		std::lock_guard<std::mutex> guard (alarm.mutex);
+		count = alarm.operations.size ();
+		sizeof_element = sizeof (decltype (alarm.operations)::value_type);
```

Comment:
Thanks for finding this, I later thought it might have needed a mutex. The other `collect_seq_con_info ()` functions declare `sizeof_element` outside of the scope guard as `value_type` is a constant known at compile time so can safely be used outside.

Reply:
Updated